src/site/content/en/fast/codelab-replace-gifs-with-video/index.md

```diff
@@ -155,11 +155,32 @@ Update the `<video>` with `<source>` elements that link to your cat-herd videos:
 
 {% Aside %}
 Browsers don't speculate about which `<source>` is optimal, so the order
-of `<source>`s matters. For example, if you specify an MP4 video first and the
+of `<source>` elements matters. For example, if you specify an MP4 video first and the
 browser supports WebM, browsers will skip the WebM `<source>` and use the MPEG-4
 instead. If you prefer a WebM `<source>` be used first, specify it first!
 {% endAside %}
 
+## Add captions and descriptions
```

Comment:
I think it's important to also note that if the gif is presentational then they should skip these steps and put role=presentation on the video. I'm a tad concerned that folks will feel like these steps are more work than alt text and they'll just bail and stick with the gif.

Reply:
How often is that likely to occur? Aren't gifs usually conveying information in some way?

I mean, using video _is_ more work, even without accessibility considerations. Do you not want this section in the codelab?

Reply:
Honestly I wish aria-label worked on video because that we match what is possible with gifs today. [Support is getting better but it's still pretty bad](https://www.powermapper.com/tests/screen-readers/media/video-aria-label/).

Maybe twitter can help: https://twitter.com/rob_dodson/status/1186446601067171845

Reply:
Based on the email convo, keeping the guidance to use description and caption tracks, even though it's a pain because `aria-label` doesn't reliably work on `<video>` elements.